.eslintrc

```diff
@@ -3,150 +3,140 @@
   "env": {
     "browser": true,
     "node": true,
-    "jasmine": true
+    "es6": true
   },
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "extends": [
+    "prettier",
+    "prettier/flowtype",
+    "prettier/react"
```

Comment:
I'm not sure actually, I've used this template in my personal libraries. It's the default template from all the repos in [react-native-community](https://github.com/react-native-community)